lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1548,4 +1549,53 @@ void amf_app_handle_initial_context_setup_rsp(
   // update UE state
   ue_context->mm_state = REGISTERED_CONNECTED;
 }
+using grpc::Status;
+int amf_app_handle_pdu_session_failure(
+    itti_n11_create_pdu_session_failure_t* pdu_session_failure) {
+  if (!pdu_session_failure) {
+    return 0;
```

Comment:
```suggestion
    return RETURNok;
```

Reply:
comment addressed

Reply:
@ganeshg87 , did you miss committing the change?

Reply:
overlooked with other fix, addressed review comment